ansible/configs/rosa-consolidated/install_rosa_sts.yml

```diff
@@ -6,6 +6,12 @@
     --mode auto
     --yes
 
+- name: Create list of AZs
+  when: rosa_classic_multi_az | bool
+  ansible.builtin.set_fact:
+    _rosa_sts_azs: |
```

Comment:
What about us-east-1 which has a-f?

Reply:
Ends up I don't need any of them.  `--multi-az` should figure it out itself.

When creating machinepools in a multi-az rosa, machines should be automatically spread between AZ.

🤞